### PR TITLE
[BPK-1249] Change label font weight

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

**Comment by matthewdavidson** [comment]
@shaundon, CSS isn't reflected in snapshots! Web components are different in that their styles are defined in css and referenced via classnames whereas react native components use inline styles.

**Comment by k0nserv** [comment]
Also added Flow type in c2143d3 which I forgot initially

**Comment by k0nserv** [comment]
I change the prop types a bit which is strictly speaking a breaking change, but then I think it wouldn't have worked if you weren't providing an element

File: packages/bpk-component-fieldset/src/BpkFieldset-test.js [hunk 1]
```diff
@@ -198,16 +199,14 @@ describe('BpkFieldset', () => {
   });
 
   it('should reject label property when label is omitted', () => {
-    expect(
-      BpkFieldset.propTypes.label({}, 'label', 'BpkFieldset').toString(),
-    ).toEqual('Error: `label` is required when `isCheckbox` is false.'); // eslint-disable-line max-len
+    expect(BpkFieldset.propTypes.label({}, 'label').toString()).toEqual(
```
Comment (matthewdavidson) [inline packages/bpk-component-fieldset/src/BpkFieldset-test.js]: Were these tests previously broken?

Comment (k0nserv) [reply packages/bpk-component-fieldset/src/BpkFieldset-test.js]: No but the extra argument shouldn't have been there

**Pull request merged**